android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -224,6 +225,25 @@ class XMTPModule : Module() {
             }
         }
 
+        AsyncFunction("sign") { clientAddress: String, message: List<Int> ->
```

Comment:
I think we assume the message is already hashed in `PrivateKey.sign`, so we should probably call the input `digest` instead of `message`.

Reply:
Perfectly fine for me to send over digests. That's how I'm doing it in JS anyways